src/components/Multisig.tsx

```diff
@@ -762,12 +764,498 @@ function AddTransactionDialog({
+        if (!multisigUsdc) {
+            const { account: multisigUsdcAcc, instructions: createmultisigUsdcItxs} = await createTokenAccount(multisigClient.provider, usdcMint, multisigPDA);
+            multisigUsdc= multisigUsdcAcc.publicKey
+            itxs.push(...createmultisigUsdcItxs);
+        }
+           // We use the uxp mint address as the seed, could use something else though.
+           const [_poolSigner] = await anchor.web3.PublicKey.findProgramAddress(
+            [uxpMint.toBuffer()],
+            UXDIDOProgramAdress
+        );
+        const poolSigner = _poolSigner;
+    // @ts-ignore
+        enqueueSnackbar("Creating USDC pool withdraw transaction", {
+            variant: "info",
+        });
+
+        const data = withdrawIdoUsdcPoolData(uxdClient);
+
+        const accounts = [
+            // HERE NEED TO ADD THE RIGHT ACCOUNTS -- Not sure what can be hardcoded or not your call for now.
+            // Accounts expected can be found here https://github.com/UXDProtocol/uxd_ido/blob/main/programs/uxd_ido/src/lib.rs#L281
+
+            // pool_account -- can be created arbitrarily - then will need to be referenced in the other operations and used on the front end (THE POINTER TO OUR IDO)
+            {
+                pubkey: new PublicKey(poolAccountAddr),
+                isWritable: true,
+                isSigner: true,
+            },
+            // pool_signer -- this is the multisig PDA : 35F3GaWyShU5N5ygYAFWDw6bGVNHnAHSe8RKzqRD2RkT
+            //? While testing we used to use a derivation from the uxp mint for creating this account
+            {
+                pubkey: poolSigner,
+                isWritable: false,
+                isSigner: true,
+            },
+
+            // pool_usdc --
+            {
+                pubkey: new PublicKey(poolUsdcAddr),
+                isWritable: false,
+                isSigner: false,
+            },
+            // distribution_authority -- should be the multisig PDA - might have an issue with the program cause it"s also a payer
+            // 35F3GaWyShU5N5ygYAFWDw6bGVNHnAHSe8RKzqRD2RkT
+            // NGMI if we discover issues last minute, this need to be done asap for our mental wellbeing, sorry to be lame but it"s super important
+            {
+                pubkey: multisigPDA,
+                isWritable: true,
+                isSigner: true,
+            },
+            // creator_usdc -- is actually the token account that will receive the usdc
+            {
+                pubkey: multisigUsdc,
+                isWritable: true,
+                isSigner: false,
+            },
+            // token_program
+            {
+                pubkey: TOKEN_PROGRAM_ID,
+                isWritable: false,
+                isSigner: false,
+            },
+
+            // clock
+            {
+                pubkey: anchor.web3.SYSVAR_CLOCK_PUBKEY,
+                isWritable: false,
+                isSigner: false,
+            },
```

Comment:
Ok now I see this is the withdraw... I shouldn't be working now, it's late. 

Not sure why the withdraw is above the initialize, that's not logical. Please move below I keep doing the same mistake since this morning lol.

Reply:
yeah I changed the order of the functions, makes more sense